src/test/java/org/jenkinsci/plugins/kubernetes/cli/KubectlBuildWrapperTest.java

```diff
@@ -140,10 +142,10 @@ public void testListingCredentialsWithoutAncestorAndMissingPermissions() throws
 
         r.jenkins.setSecurityRealm(r.createDummySecurityRealm());
         ProjectMatrixAuthorizationStrategy as = new ProjectMatrixAuthorizationStrategy();
-        as.add(Jenkins.READ, "user-not-enough-permissions");
+        as.add(Jenkins.READ, new PermissionEntry(AuthorizationType.EITHER, "user-not-enough-permissions"));
         r.jenkins.setAuthorizationStrategy(as);
 
-        try (ACLContext unused = ACL.as(User.get("user-not-enough-permissions", true, null).impersonate())) {
+        try (ACLContext unused = ACL.as2(User.get("user-not-enough-permissions", true, null).impersonate2())) {
```

Comment:
For myself: https://javadoc.jenkins-ci.org/hudson/security/ACL.html#impersonate2(org.springframework.security.core.Authentication)